clusterloader2/testing/access-tokens/config.yaml

```diff
@@ -42,7 +42,11 @@
 {{$RESTART_COUNT_THRESHOLD_OVERRIDES:= DefaultParam .RESTART_COUNT_THRESHOLD_OVERRIDES ""}}
 
 name: access-tokens
-automanagedNamespaces: {{$namespaces}}
+namespace:
+  count: {{$namespaces}}
+  prefix:
```

Comment:
With the defaulting you added, neither of those below should be needed (you only need to specify count).

Reply:
done